viewing_party/party.py

```diff
@@ -1,23 +1,174 @@
+"""
+Dear Ashley (or Instructor),
+
+Can you please take a look at the comments in my wave 3.
+I tried doing something that didn't work out and so had to 
+abandon it all together. I can't figure out what the problem
+is and would appreciate a look.
+
+Also, answering my questions on Wednesday!
+And WOOT WOOT for finishing something I thought would even 
+be able to do halfway!!!!!
+
+Thank you so much,
+Huma Hameed
+(she/her)
+"""
+
+
+
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {}
+    new_movie["title"] = title
+    new_movie["genre"] = genre
+    new_movie["rating"] = rating
+
+    for key, value in new_movie.items():
+        print(new_movie)
+        if value == None:
+            return None
+    return new_movie
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"] = []
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    # for movies["title"] in user_data["watched"] == title:
+    for movies in user_data["watchlist"]:
+        if movies["title"] == title:
+            print(title)
+            user_data["watched"].append(movies)
+            user_data["watchlist"].remove(movies)
+    return user_data

```

Comment:
Here I might suggest that you actually add a break in your if block.

Note that, while we might only expect there to be one movie with a particular title in the watchlist (might not be a good assumption, but for this simplified data, it might be ok), this implementation is fine. 

As soon as we make a change (like removing from watchlist) we want to stop iterating, so we use break. By adding break, the loop iterates the watchlist, and when the desired movie is found, adds it to the watched list and removes it from the watchlist. If we returned immediately at this point, everything would be fine!

Without the break statement, we might be in a scenario where title actually matches to more than one movie and see some weirdness.